src/main/java/com/three_tech_solutions/slot_app/data/models/Student.java

```diff
@@ -23,18 +21,24 @@ public class Student {
     private String pathologies;
     private LocalDate admissionDate;
     private boolean enabled;
-    @OneToOne
+    @OneToOne(cascade = CascadeType.ALL)
     private Plan plan;
     @OneToMany
     private List<Payment> payments;
-    @Id
-    private UUID id = UUID.randomUUID();
     @ManyToOne
     private User user; // Relación con el profesional que registró al alumno
```

Comment:
Este comentario lo podrias borrar

```suggestion
    private User user;
```